charts/gardener/gardenlet/values.yaml

```diff
@@ -82,6 +82,7 @@ global:
         shootCare:
           concurrentSyncs: 5
           syncPeriod: 30s
+          staleExtensionHealthCheckThreshold: 5m
```

Comment:
I think you also have to adapt https://github.com/gardener/gardener/blob/master/charts/gardener/gardenlet/charts/runtime/templates/configmap-componentconfig.yaml#L91-L97

Reply:
Thanks. 
I also saw that the retrySyncPeriod while still being specified in the Gardenlet config and helm chart, is not in use by Gardener any more -> removed it it. Does that make sense?

Reply:
Yes, makes sense. Maybe a separate PR?

Reply:
OK, as it is now in already let's keep it in.

Reply:
Can you add a proper release note for the removal?

Reply:
Can you also remove it from https://github.com/gardener/gardener/pull/2215/files#diff-25a02c1ffdae2994b5ee73e00fbd98d9R79?

Reply:
Can you send the link to the file again? I am not sure where.

Reply:
IN the gardenlet chart values, just a few lines above in the `shoot:` section